cmd/promtool/tsdb.go

```diff
@@ -617,8 +571,20 @@ func analyzeBlock(b tsdb.BlockReader, limit int) error {
 	return nil
 }
 
-func dumpSamples(db *tsdb.DBReadOnly, mint, maxt int64) (err error) {
+func DumpSamples(path string, mint, maxt int64) error {
```

Comment:
Why do you think it needs this extra func?

Reply:
Done